AUTHORS

```diff
@@ -26,9 +26,10 @@ David Martos <davidmartos96@gmail.com>
 Juan Pablo Paulsen <juanpablo.paulsen@gmail.com>
 Luke Pighetti <git@lukepighetti.com>
 Sem van Rij <sem@lekkerplakjekaas.nl>
+Michael Joseph Rosenthal <rosenthalm93@gmail.com>
 Muhammed Salih Guler <muhammedsalihguler@gmail.com>
 Nolan Scobie <nolan.scobie@gmail.com>
 Cameron Steffen <cam.steffen94@gmail.com>
-Danny Tuppeny <danny@tuppeny.com>
+Danny Tuppeny <danny@tuppeny.com> 
```

Comment:
Accidental space. Probably doesn't matter, but it's there.

Reply:
Fixed.  Thanks!